packages/utils/src/coverage-assessment.ts

```diff
@@ -62,12 +66,15 @@ async function main() {
 
   const startedAt = Date.now();
   log(`>>> Starting with ${patientIds.length} patient IDs...`);
+
   const { orgName } = await getCxData(cxId, undefined, false);
-  const fileName = csvName(orgName);
+  const { patientIds: patientIdsToQuery, isAllPatients } = await getPatientIds();
 
+  await displayWarningAndConfirmation(patientIdsToQuery.length, isAllPatients, log);
+
+  const fileName = csvName(orgName);
```

Comment:
this `csvName` / `initCsv` block is after the warning in this script, but before the warning in the other script. Any reason for the switch? Otherwise I'd pick one and make them consistent (will helper with abstraction later if we decide to move shared functionality into another location).

Reply:
No particular reason, moved it around and both `bulk-query-patients.ts` and `coverage-assessment.ts` have a pretty similar initialization.